src/libtorch.cc

```diff
@@ -532,15 +532,17 @@ ModelState::ParseParameters()
         TRITONSERVER_ErrorDelete(err);
       }
     } else {
-      if (inter_op_thread_count > 0) {
+      if ((inter_op_thread_count > 0) &&
+          (inter_op_thread_count != at::get_num_interop_threads()) &&
```

Comment:
This can still allow a repeated call if two model config files set the number of interop threads to different values. (See #159 for a safer solution.)

Reply:
Good. thanks for fixing the multiple config file scenario as well!